src/main/java/org/dhatim/businesshours/BusinessPeriod.java

```diff
@@ -83,6 +83,15 @@ public CronExpression getStartCron() {
         return alwaysOpen() ? null : new CronExpression(start);
     }
 
+    /**
+     * Get a {@link CronExpression} that triggers at each period closing.
+     * e.g. if the period is 9am-18pm, the result will be <code>59 18 * * *</code>
+     * @return <code>null</code> if the period is always open, else the cron expression
+     */
+    public CronExpression getEndCron() {
+        return alwaysOpen() ? null : new CronExpression(end);

```

Comment:
I think it should rather be `new CronExpression(end.increment())`, because `end` is included in the business period (the closing cron of `wday{Mon-Fri} hr{9-18}` should be `0 19 * * 1-5` and not `59 18 * * 1-5`).

Reply:
Good idea to get the next value.
I update it and